internal/util/projutil/project_util.go

```diff
@@ -207,11 +207,31 @@ func GetOperatorType() OperatorType {
 	return OperatorTypeUnknown
 }
 
+// PluginKeyToOperatorType converts a plugin key string to an operator project
+// type.
+// TODO(estroz): this can probably be made more robust by checking known
```

Comment:
remove this TODO?

Reply:
Hi @tengqm, 

We do not change the implementation of this method in the PR, so the TODO still in place for we are able to check it further in the future.

Reply:
Marking as solved.